src/getdim.f

```diff
@@ -1,33 +1,24 @@
+!>    @file
+!>    @brief This routine converts grid information from a GRIB2 grid
```

Comment:
@Hang-Lei-NOAA I don't see any references to GRIB1 in this routine. Is this description correct?

Reply:
This was not. updated